### PR TITLE
tileSize is no longer hard corded in Zoomify Tile Source

### PR DESCRIPTION
**Pull request by abrlam**
minor fix as per this issue:
https://github.com/openseadragon/openseadragon/issues/1704#issuecomment-529580041

### CODE REVIEW

File: src/zoomifytilesource.js [hunk 1]
```diff
@@ -44,7 +44,9 @@
      * @param {String} tilesUrl
      */
     $.ZoomifyTileSource = function(options) {
-        options.tileSize = 256;
+        if(options.tileSize === undefined){
```
Comment (msalsbery) [inline src/zoomifytilesource.js:47]: Would you mind using
```
typeof options.tileSize === ‘undefined’
```
instead here?

**Comment by abrlam** [comment]
Certainly. Let me know if further changes are needed. Thank you again.

**Comment by msalsbery** [comment]
> Certainly. Let me know if further changes are needed. Thank you again.

@abrlam  Great, thank you!

One more item...could you please add an entry to changelog.txt (add to the list under "2.4.3: (In progress)")? Please mention "#1704" in your description...this helps us more easily find when/where things changed.

Thanks for the patch!!

@iangilman Any further review?

**Comment by abrlam** [comment]
Not a problem @msalsbery. I updated the change log as well. Thank you again.

**Comment by abrlam** [comment]
@iangilman Yes I did.

Here's the test link (tileSize is set as 512):
http://testing30052020.imfast.io/osd_zoomify_tilesize_fix/zoomify.html

**Comment by bluehaoran** [comment]
Thanks for fixing it!

I'll have to dig around for our original tiled images (or regenerate some new ones).... I'll get back to you and/or re-open the bug if it's not fixed, but this looks like a correct fix as far as I'm concerned!

**Comment by iangilman** [comment]
Awesome. Thank you all for contributing to this patch! :-)

**Pull request merged**